cloudferrylib/utils/drivers/ssh_chunks.py

```diff
@@ -51,128 +54,129 @@ def transfer(self, data):
         ssh_user_dst = self.cfg.dst.ssh_user
         ssh_sudo_pass_dst = self.cfg.dst.ssh_sudo_password
 
-        src_temp_dir = os.path.join(self.cfg.src.temp, '')
-        dst_temp_dir = os.path.join(self.cfg.dst.temp, '')
+        runner_src = remote_runner.RemoteRunner(host_src, ssh_user_src)
+        runner_dst = remote_runner.RemoteRunner(host_dst, ssh_user_dst)
+        temp_dir_name = os.popen('mktemp -udt ssh_chunks_XXXX').read().rstrip()
```

Comment:
I mean `-u` key for `mktemp` command. Here is `-udt`, but in `cloudferrylib/os/actions/check_bandwidth.py` - only `-dt`. I am interested, maybe it is spesial reason for it?